VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -107,12 +110,15 @@ public void run() {
         DiscussionTextUtils.setEndorsedState(authorLayoutViewHolder.answerTextView,
                 discussionThread, discussionResponse);
 
+        textViewAddComment.setText(R.string.discussion_add_comment_button_label);
         buttonAddComment.setOnClickListener(new View.OnClickListener() {
             public void onClick(View v) {
                 createComment();
             }
         });
         buttonAddComment.setEnabled(false);
+        buttonAddComment.setContentDescription(getString(R.string.discussion_add_comment_button_description));
+        editTextNewComment.setHint(R.string.discussion_add_comment_hint);
```

Comment:
Not sure while using databinding, how can I detect which fragment are using ? So, I guess using data binding for this case seems like a NO.